resources/views/layouts/app.blade.php

```diff
@@ -12,9 +12,14 @@
 
     <!-- Styles -->
     <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+    <link href="{{ asset('css/sidebar.css') }}" rel="stylesheet" />
+
 
-    <link rel="stylesheet" href="{{ asset('css/bootstrap/bootstrap-theme.min.css')}}">
```

Comment:
did you remove this because of the error in the browser? dont remove this, import the glyphicons instead inside the css folder